OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -34,6 +34,10 @@ public class BuildingInfo : ITraitInfo, IOccupySpaceInfo, IPlaceBuildingDecorati
 		public readonly CVec Dimensions = new CVec(1, 1);
 		public readonly bool RequiresBaseProvider = false;
 		public readonly bool AllowInvalidPlacement = false;
+		[Desc("clear smudges before add building")]
```

Comment:
These are sentences, so should start with a capital letter and end with a full stop.  It would also be useful to mention that they apply to the building footprint.

e.g. "Clear smudges from underneath the building footprint."